x-pack/plugins/security_solution/server/config.ts

```diff
@@ -109,6 +109,19 @@ export const configSchema = schema.object({
    * Artifacts Configuration
    */
   packagerTaskInterval: schema.string({ defaultValue: '60s' }),
+
+  /**
+   * For internal use. Specify which version of the Detection Rules fleet package to install
+   * when upgrading rules. If not provided, the latest compatible package will be installed,
+   * or if running from a dev environment or -SNAPSHOT build, the latest pre-release package
+   * will be used (if fleet is available or not within an airgapped environment).
+   *
+   * Note: This is for `upgrade only`, which occurs by means of the `useUpgradeSecurityPackages`
+   * hook when navigating to a Security Solution page. The package version specified in
+   * `fleet_packages.json` in project root will always be installed first on Kibana start if
+   * the package is not already installed.
+   */
+  prebuiltRulesPackageVersion: schema.maybe(schema.string()),
```

Comment:
Nit: `schema.string({ defaultValue: '' })` could simplify the setting's type -- just personally prefer empty strings to something like `string | null | undefined`. But that's a very subjective comment so please ignore it if you will.

Reply:
I had it this way at first...but then swapped to `undefined` as default to make it clearer what the empty-value was if coming across this config elsewhere in code 😅 (since the plugin API docs don't seem to follow this prop around in the IDE).